addons/fieldmanual/CfgHintCategories.hpp

```diff
@@ -0,0 +1,9 @@
+#include "script_component.hpp"
+
+class CfgHintCategories
+{
+    class ACE_FieldManual
+    {
+        displayName = CSTRING(HintCategory_FieldManual_DisplayName);
+    };
+};
```

Comment:
Review coding style for brace placement, fix across all files that are incorrect
https://ace3.acemod.org/wiki/development/coding-guidelines#5-code-style

Reply:
I'll get working on it